chunky/src/java/se/llbit/chunky/world/biome/Biomes.java

```diff
@@ -38,97 +40,117 @@
 
   public static final Biome unknown = new Biome("", "unknown", 0.5, 0.5, 0x7E7E7E, 0x7E7E7E, 0x7E7E7E);
 
+  public static final Map<String, Biome> biomesByResourceLocation = new Object2ReferenceOpenHashMap<>();
+  public static final Object2IntMap<String> biomeIDsByResourceLocation = new Object2IntOpenHashMap<>();
+
+  public static final List<Biome> biomes = new ArrayList<>();
+
   // The fallback foliage and grass colors of the biomes were calculated with the default resourcepack.
   // The map colors use the default biome colors by Amidst, see https://github.com/toolbox4minecraft/amidst/wiki/Biome-Color-Table.
```

Comment:
It took a few minutes of digging through my hard drive to find the generator script. We should add it to this comment :sweat_smile:

Reply:
Also, would you mind PRing the updates to it?